pkgs/development/tools/misc/kafdrop/default.nix

```
@@ -0,0 +1,56 @@
+{ stdenv, maven, fetchFromGitHub, makeWrapper, jre, lib }:
+let
+  version = "3.29.0";
+
+  src = fetchFromGitHub {
+    owner = "obsidiandynamics";
+    repo = "kafdrop";
+    rev = "${version}";
+    sha256 = "sha256-38ysop/WcV+eHJCdzryTZOdtik6vfSfS5hy5RYRfAFw=";
+  };
+
+  deps = stdenv.mkDerivation {
+    name = "kafdrop-${version}-deps";
+    inherit src;
+    buildInputs = [ maven ];
+    buildPhase = ''
+       mvn package -Dmaven.repo.local=$out
+    '';
+    installPhase = ''
+    find $out -type f \
+      -name \*.lastUpdated -or \
+      -name resolver-status.properties -or \
+      -name _remote.repositories \
+      -delete
+  '';
+    dontFixup = true;
+    outputHashAlgo = "sha256";
+    outputHashMode = "recursive";
+    outputHash = "sha256-88pxb9ih0PA9jp13QbycR0w9e7ubT6lRdC3VnrKzG6U=";
+  };
+
+in
+stdenv.mkDerivation rec {
+  pname = "kafdrop";
+  inherit version;
+  inherit src;
+  buildInputs = [ maven makeWrapper ];
+  buildPhase = ''
+    echo "Using repository ${deps}"
+    mvn --offline -Dmaven.repo.local=${deps} package;
+  '';
+  installPhase =''
+    mkdir -p $out/bin
+    mkdir -p $out/share/java
+    mv target/${pname}-${version}.jar $out/share/java/
+    makeWrapper ${jre}/bin/java $out/bin/${pname} \
+        --add-flags "-jar $out/share/java/${pname}-${version}.jar"
+                '';
```

```suggestion
      --add-flags "-jar $out/share/java/${pname}-${version}.jar"
    '';
```
We use 2 space indentation.